test/testutil/apps_mem.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
??? shouldn't this just be 2019 ???

Reply:
copying and pasting, you know...  Fixing